src/client/java/teammates/client/scripts/DataMigrationForOrphanedStudentProfile.java

```diff
@@ -0,0 +1,45 @@
+package teammates.client.scripts;
+
+import java.io.IOException;
+
+import com.googlecode.objectify.Key;
+import com.googlecode.objectify.cmd.Query;
+
+import teammates.storage.entity.StudentProfile;
+
+/**
+ * Script to delete all orphaned StudentProfile entity.
+ */
+public class DataMigrationForOrphanedStudentProfile extends DataMigrationWithCheckpointForEntities<StudentProfile> {
+
+    public static void main(String[] args) throws IOException {
+        new DataMigrationForOrphanedStudentProfile().doOperationRemotely();
+    }
+
+    @Override
+    protected Query<StudentProfile> getFilterQuery() {
+        return ofy().load().type(StudentProfile.class);
+    }
+
+    @Override
+    protected String getLastPositionOfCursor() {
+        return "";
+    }
+
+    @Override
+    protected boolean isPreview() {
+        return true;
```

Comment:
Should `isPreview` and `lastPositionOfCursor` be passed in as command line arguments instead so that we don't need to change the source code and recompile?

Reply:
I think the script is always run in IDE and it is the way [recommended](https://github.com/TEAMMATES/teammates/blob/master/docs/development.md#running-client-scripts)?

Reply:
You can still pass command line arguments when running in the IDE. My point is to eliminate the recompilation of the program. For this simple program it doesn't make a difference

Reply:
I agree that by right such parameter should be in command line or configuration file. But consider we usually run the script in IDE, it would be more convenient for the Admin to change the code than to change the run configuration.